### PR TITLE
chore: import translations for hu (manual)

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Import Crowdin translations.

Thank you to everyone contributing to translate ethereum.org ❤️

## Content buckets imported
3, 6, 7, 10, 11, 12, 13, 16, 21

## Preview link
- https://deploy-preview-13113--ethereumorg.netlify.app/hu/

PR replaces #13085

### CODE REVIEW

**Comment by wackerow** [comment]
Note to self: use `crowdin` in branch name for any manual imports to avoid unnecessary check failure

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ A blokk `body` számos mezőt tartalmaz:
 | `graffiti`           | tetszőleges adat a blokkok taggelésére                              |
 | `proposer_slashings` | a validátorok listája, akiket slashelni kell                        |
 | `attester_slashings` | a tanusítók listája, akiket slashelni kell                          |
-| `attestations`       | a tanúsítók listája, akik ezt a blokkot támogatják                  |
-| `deposits`           | az új letétek listája a letéti szerződésbe                          |
+| `tanúsítások`        | a tanúsítók listája, akik ezt a blokkot támogatják                  |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:59]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 2]
```diff
@@ -56,8 +56,8 @@ A blokk `body` számos mezőt tartalmaz:
...
+| `behelyezés`         | az új letétek listája a letéti szerződésbe                          |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:60]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 3]
```diff
@@ -67,8 +67,8 @@ A `attestations` (tanúsítások) mező tartalmazza a blokkban lévő az összes
 | Mező               | Leírás                                                       |
 |:------------------ |:------------------------------------------------------------ |
 | `aggregation_bits` | a validátorok listája, akik részt vettek a tanúsításban      |
-| `data`             | konténer számos almezővel                                    |
-| `signature`        | az összes tanúsítást végző validátor aláírásának aggregátuma |
+| `adat`             | konténer számos almezővel                                    |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:70]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 4]
```diff
@@ -67,8 +67,8 @@ A `attestations` (tanúsítások) mező tartalmazza a blokkban lévő az összes
...
+| `aláírás`          | az összes tanúsítást végző validátor aláírásának aggregátuma |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:71]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 5]
```diff
@@ -95,7 +95,7 @@ Az `execution_payload_header` a következő mezőket tartalmazza:
 | `block_number`      | a jelenlegi blokk száma                                                  |
 | `gas_limit`         | a jelenlegi blokkban megengedett maximális gáz                           |
 | `gas_used`          | a jelenlegi blokkban elhasznált gáz aktuális összege                     |
-| `timestamp`         | a blokk ideje                                                            |
+| `időbélyeg`         | a blokk ideje                                                            |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:98]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 6]
```diff
@@ -115,12 +115,12 @@ Az `execution_payload` maga a következőket tartalmazza (ami azonos a fejlécce
 | `block_number`     | a jelenlegi blokk száma                                                  |
 | `gas_limit`        | a jelenlegi blokkban megengedett maximális gáz                           |
 | `gas_used`         | a jelenlegi blokkban elhasznált gáz aktuális összege                     |
-| `timestamp`        | a blokk ideje                                                            |
+| `időbélyeg`        | a blokk ideje                                                            |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:118]: seems these words shouldn't be translated

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 7]
```diff
@@ -115,12 +115,12 @@ Az `execution_payload` maga a következőket tartalmazza (ami azonos a fejlécce
...
 | `extra_data`       | tetszőleges hozzáadott adat, mint nyers bájt                             |
 | `base_fee_per_gas` | az alapdíj értéke                                                        |
 | `block_hash`       | a végrehajtó blokk hash-e                                                |
 | `tranzakciók`      | a tranzakciók listája, amit végre kell hajtani                           |
-| `withdrawals`      | a visszavonásra kerülő objektumok listája                                |
+| `kivételek`        | a visszavonásra kerülő objektumok listája                                |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:123]: seems these words shouldn't be translated

**Review by nhsz — CHANGES_REQUESTED** [review]
some terms are being translated but shouldn't

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 1 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:59]:
```suggestion
| `attestations`        | a tanúsítók listája, akik ezt a blokkot támogatják                  |
```

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 2 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:60]:
```suggestion
| `deposits`         | az új letétek listája a letéti szerződésbe                          |
```

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 4 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:71]:
```suggestion
| `data`             | konténer számos almezővel                                    |
| `signature`          | az összes tanúsítást végző validátor aláírásának aggregátuma |
```

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 5 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:98]:
```suggestion
| `timestamp`         | a blokk ideje                                                            |
```

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 6 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:118]:
```suggestion
| `timestamp`        | a blokk ideje                                                            |
```

File: public/content/translations/hu/developers/docs/blocks/index.md (hunk 7 above)
Comment (wackerow) [inline public/content/translations/hu/developers/docs/blocks/index.md:123]:
```suggestion
| `withdrawals`        | a visszavonásra kerülő objektumok listája                                |
```

File: public/content/translations/hu/developers/docs/blocks/index.md [hunk 8]
```diff
@@ -45,7 +45,7 @@ A blokkban rengeteg információ van. A legmagasabb szinten a következő mezők
 | `proposer_index` | a validátor azonosítója, aki a blokkot javasolta                              |
 | `parent_root`    | az előző blokk hash-e                                                         |
 | `state_root`     | a státusz objektum gyökér hash-e                                              |
-| `body`           | egy olyan objektum, amely számos mezőt tartalmaz, ahogy azt alább definiáljuk |
+| `törzs`          | egy olyan objektum, amely számos mezőt tartalmaz, ahogy azt alább definiáljuk |
```
Comment (nhsz) [inline public/content/translations/hu/developers/docs/blocks/index.md:48]:
```suggestion
| `body`           | egy olyan objektum, amely számos mezőt tartalmaz, ahogy azt alább definiáljuk |
```

**Review by nhsz — APPROVED** [review]

**Pull request merged**